### DISCUSSION TITLE
Where are clear instructions for initial setup of the mobile app?

### DISCUSSION THREAD

**Discussion by indeedwatson** [zeapo/Android-Password-Store#2331] [Q&A]
Every time I need to set up APS on a new mobile device I'm utterly confused. I eventually manage to do it, but it's never clear to me exactly what are the steps I need to follow. Here's my current attempt:

Installed APB and Openkeychain. 

Exported my secret gpg key from my desktop to OpenKeychain.

Opened APB, add a repository, enter the server URL, select OPENKEYCHAIN, hit clone.

I get a popup to select an authentication key. There are none, there's no keys listed.

What do I do now? Is there something I'm missing or doing wrong?

**Comment by msfjarvis** [comment]
If the key is not suggested then it is likely missing an authentication subkey. You can check its subkeys in OpenKeychain to confirm that.

**Reply by indeedwatson** [reply]
There's 2 subkeys: one has a seal and a pen icon, and the other has a lock icon.

**Reply by BrianENguyen** [reply]
I'm in the same situation as you right now. I'm assuming you've figured this out already? If so then any guidance would be greatly appreciated!
